src/sql/traits.py

```diff
@@ -0,0 +1,54 @@
+from traitlets import TraitError, TraitType
+from sql import display
+import warnings
+
+VALUE_WARNING = (
+    'Please use a valid option: "warn", "enabled", or "disabled". \n'
+    "For more information, "
+    "see the docs: https://jupysql.ploomber.io/en/latest/api/configuration.html"
```

Comment:
add `#named-parameters` to the link so users are taken to a specific section

Reply:
[Added #named-parameters](https://github.com/ploomber/jupysql/pull/980/files#diff-386de3bc0fe97ed1ed445b5b7535ff54a341ced84c18cf4daa7327742a894964R9)